websites/T/Tempest Fansub/presence.ts

```diff
@@ -0,0 +1,124 @@
+const presence = new Presence({
+		clientId: "1047102386478534727",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const { pathname } = window.location,
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/bhEsT7P.png",
+		};
+	let url: string;
+	if (location.origin.startsWith("https://"))
+		url = location.origin.replace("https://", "");
+	else url = location.origin;
```

Comment:
[`origin` ](https://developer.mozilla.org/en-US/docs/Web/API/Location/origin) always starts with the protocol. If you are trying to use a property without the protocol, check out [`hostname`](https://developer.mozilla.org/en-US/docs/Web/API/Location/hostname) or [`host`](https://developer.mozilla.org/en-US/docs/Web/API/Location/host)